src/ServiceControl/Notifications/Email/EmailSender.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace ServiceControl.Notifications.Email
+{
+    using System;
+    using System.Net;
+    using System.Net.Mail;
+    using System.Threading.Tasks;
+    using NServiceBus.Logging;
+
+    class EmailSender
+    {
+        public static async Task Send(EmailNotifications settings, string subject, string body, string emailDropFolder = null)
+        {
+            try
+            {
+                using (var client = CreateSmtpClient(settings, emailDropFolder))
+                {
+                    using (var mailMessage = new MailMessage(settings.From, settings.To, subject, body))
```

Comment:
using could be on the same level as outer to reduce nesting for better readability

Reply:
done